### DISCUSSION TITLE
My fixture files gets updated in Before and After hook, Cypress test runner feels that there is a change and reruns the test.

### DISCUSSION THREAD

**Discussion by Mustafahameed96** [cypress-io/cypress#15861] [Questions and Help]
My fixture files gets updated in Before and After hook, Cypress test runner feels that there is a change and reruns the test.

**Comment by bahmutov** [comment]
I am not sure what is the problem or the expected vs actual behavior here. Please consult our documentation, as it extensively covers the test organization https://on.cypress.io/writing-and-organizing-tests
